src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -1167,7 +1167,7 @@ static bool monitors_used_above_threshold(MonitorList* list) {
       _no_progress_cnt >= NoAsyncDeflationProgressMax) {
     float remainder = (100.0 - MonitorUsedDeflationThreshold) / 100.0;
     size_t new_ceiling = ceiling + (ceiling * remainder) + 1;
-    size_t old_ceiling = ObjectSynchronizer::in_use_list_ceiling();
+    size_t old_ceiling = ceiling;
```

Comment:
Is that really what you want? `ceiling` may have been assigned again at line 1160.

Reply:
I misread your comment... Fixing it now...